Stream-Two/Unit03-Testing/challenges/challenge1/guess_the_number.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+NUMBER_OF_GUESSES = 10
+RANGE_TOP = 10
+RANGE_BOTTOM = 0
+"""
```

Comment:
This should be a regular comment - docstring comments are only supported as the first element of a module, function or class